scripts/build.js

```diff
@@ -0,0 +1,110 @@
+const fs = require('fs');
+const path = require('path');
+
+const { ucs2 } = require("punycode"); 
+const SimpleIcons = require('simple-icons');
+const svg2ttf = require('svg2ttf');
+const SVGPath = require('svgpath');
+const ttf2woff = require('ttf2woff');
+const ttf2woff2 = require('ttf2woff2');
+
+
+const DISTDIR = path.resolve(__dirname, '..', 'font');
+
+const buildSimpleIconsSvgFontFile = () => {
+  let fileContent = '<?xml version="1.0" standalone="no"?>\n'
+                  + '<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">\n'
+                  + '<svg xmlns="http://www.w3.org/2000/svg">\n'
+                  + '<metadata>CC0 1.0 Universal | simple-icons contributors</metadata>\n'
+                  + '<defs>\n'
+                  + '<font id="Simple Icons" horiz-adv-x="1200">\n'
+                  + '<font-face font-family="Simple Icons" units-per-em="1200" ascent="-1" descent="1200" />\n'
+                  + '<missing-glyph horiz-adv-x="0" />\n';
```

Comment:
How about :arrow_down: ?

```suggestion
  let fileContent = `<?xml version="1.0" standalone="no"?>
                  <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
                  <svg xmlns="http://www.w3.org/2000/svg">
                  <metadata>CC0 1.0 Universal | simple-icons contributors</metadata>
                  <defs>
                  <font id="Simple Icons" horiz-adv-x="1200">
                  <font-face font-family="Simple Icons" units-per-em="1200" ascent="-1" descent="1200"/>
                  <missing-glyph horiz-adv-x="0"/>`;
```